common/realtime.py

```diff
@@ -49,14 +49,14 @@ def config_realtime_process(core: int, priority: int) -> None:
 
 
 class Ratekeeper:
-  def __init__(self, rate: int, print_delay_threshold: Optional[float] = 0.0) -> None:
+  def __init__(self, rate: int, process_name: str = "-", print_delay_threshold: Optional[float] = 0.0) -> None:
     """Rate in Hz for ratekeeping. print_delay_threshold must be nonnegative."""
     self._interval = 1. / rate
     self._next_frame_time = sec_since_boot() + self._interval
     self._print_delay_threshold = print_delay_threshold
     self._frame = 0
     self._remaining = 0.0
-    self._process_name = multiprocessing.current_process().name
+    self._process_name = multiprocessing.current_process().name + f" '{process_name}'"
```

Comment:
`multiprocessing.current_process().name` doesn't cover this?

Reply:
The name 'Process-1' didn't seem that useful to me.

Reply:
instead of passing it in, let's replace `multiprocessing.current_process().name` with `getproctitle`. manager already sets the proc title https://github.com/commaai/openpilot/blob/acd455ed3ae0ffbb82fe36dd6d9a1f4d16279823/selfdrive/manager/process.py#L32

Reply:
Great! That looks much better.